components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -725,4 +730,27 @@ private JWTClaimsSet handleTokenBinding(JWTClaimsSet.Builder jwtClaimsSetBuilder
         }
         return jwtClaimsSetBuilder.build();
     }
+
+    /**
+     * Set signed tenant domain to the JWT token's realm claim.
+     * @param tenantDomain
+     * @param jwtClaimsSet
+     * @return
+     */
+    private JWTClaimsSet setSignerRealm(String tenantDomain, JWTClaimsSet jwtClaimsSet) {
+
+        Map<String, String> realm = new HashMap<>();
+        realm.put(OAuthConstants.OIDCClaims.TENANT, tenantDomain);
+
+        if (log.isDebugEnabled()) {
+            log.debug("Setting signer tenant domain : " + tenantDomain + " to the 'realm' claim of " +
```

Comment:
seems the debug message is not clearly understandable shall we rephrase it?

Reply:
Updated the message.